github.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/google/go-github/github"
 )
 
+var XYZ = 3
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var XYZ should have comment or be unexported